### PR TITLE
FlutterNinjas 2025

### PR DESCRIPTION
**Pull request by dkbast**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ If you want to add a conference, feel encouraged to create a PR which adds the c
 | Name                            | Date                             | CfP-Link                   | CfP-Deadline-Date   | Place                  | Aprox. Attendees |
 | ------------------------------- | -------------------------------- | -------------------------- | ------------------- | ---------------------- | ---------------- |
 | [App Devcon Amsterdam][36] 2025 | March 18th - 21th, 2025          | [App Devcon Amsterdam][37] | December 20th, 2024 | Amsterdam, Netherlands | 1000+            |
+| [FlutterNinjas][5] 2024         | May 29th - 30th, 2025            | [FlutterNinjas CfP][40     | March 2nd, 2025     | Tokyo, Japan           | ???              |
```
Comment (AlexanderThiele) [inline README.md:12]: i think there is a ] missing at `[FlutterNinjas CfP][40`

**Review by AlexanderThiele — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dkbast) [inline README.md:12]:
```suggestion
| [FlutterNinjas][5] 2024         | May 29th - 30th, 2025            | [FlutterNinjas CfP][40]     | March 2nd, 2025     | Tokyo, Japan           | ???              |
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ If you want to add a conference, feel encouraged to create a PR which adds the c
 | Name                            | Date                             | CfP-Link                   | CfP-Deadline-Date   | Place                  | Aprox. Attendees |
 | ------------------------------- | -------------------------------- | -------------------------- | ------------------- | ---------------------- | ---------------- |
 | [App Devcon Amsterdam][36] 2025 | March 18th - 21th, 2025          | [App Devcon Amsterdam][37] | December 20th, 2024 | Amsterdam, Netherlands | 1000+            |
+| [FlutterNinjas][5] 2024         | May 29th - 30th, 2025            | [FlutterNinjas CfP][40]     | March 2nd, 2025     | Tokyo, Japan           | ???              |
```
Comment (spydon) [inline README.md:12]:
```suggestion
| [FlutterNinjas][5] 2024         | May 29th - 30th, 2025            | [FlutterNinjas CfP][40]    | March 2nd, 2025     | Tokyo, Japan           | ???              |
```

**Review by AlexanderThiele — APPROVED** [review]
looks good

**Pull request merged**